add two FAQs concerning the hackathon (#28)
felvin-search/docs: docs/events/hackathon.mdx

- [Overview of Getting Started with Instant Apps](../instant-apps/getting-started.md)
- [GitHub README and contributing guide](https://github.com/felvin-search/instant-apps)
 

> ### Do the contributions I make during hacktoberfest count?

Reviewer: ```suggestion
### Do the contributions I make during hacktoberfest (outside the hackathon dates) count?
```

> - No. Only those made during the hackathon period (15-17) will count.

Reviewer: ```suggestion
- No. Only the Pull Requests made during the hackathon period (15-17 October) will count.
```

> ### Will submissions on existing issues count?

Reviewer: ```suggestion
### Can I pick up existing issues in the repository and work on them or do I have to think about new instant app ideas?
```

> - Yes. But make sure you say you are working on an issue to avoid submissions of multiple pull requests.

Reviewer: ```suggestion
- Yes - you can work on existing issues. Do make sure to comment and say you are working on an issue to avoid submission of multiple pull requests. However, we do encourage thinking of new Instant Apps which could benefit your search experience. :)
```
